### ISSUE TITLE
Couldn't start project on Android: Error running adb

### ISSUE THREAD

**Issue by SandroFrazaoS** [expo/expo-cli#2741]
Não estou conseguindo emular o aplicativo no computador.
a mensagem de erro que ocorre é; 

Couldn't start project on Android: Error running adb: The emulator (Pixel_2_API_29) quit before it finished opening. You can try starting the emulator manually from the terminal with: C:\Users\sandr\AppData\Local\Android\Sdk/emulator/emulator @Pixel_2_API_29

reinstalei o andoid studio 2x
criei outros virtuais devices = Pixel_2_API_28, Pixel_2_API_30 mas ocorre o mesmo erro.

segue as telas dos passos que executo ate o erro e configuração da maquina

configuração da maquina
![Screenshot_1](https://user-images.githubusercontent.com/54441280/94753511-d203a800-0364-11eb-98c4-8361a9e4d250.png)

erro no androide studio quanto tento executar
![Screenshot_2](https://user-images.githubusercontent.com/54441280/94753514-d334d500-0364-11eb-8abf-55dc33f6b6e9.png)

executo o expo start pelo powershell
![Screenshot_5](https://user-images.githubusercontent.com/54441280/94753529-d8921f80-0364-11eb-9ff2-4b738606b264.png)

depois de carregar o programa ao clicar no 'Run on Androide device/emulator' ocorre o erro
![Screenshot_6](https://user-images.githubusercontent.com/54441280/94753530-d92ab600-0364-11eb-859a-051fb6d986c2.png)

erro no androide studio
![Screenshot_7](https://user-images.githubusercontent.com/54441280/94753533-da5be300-0364-11eb-996c-3197d1998fb0.png)

configuração utilizado no androide studio
![Screenshot_3](https://user-images.githubusercontent.com/54441280/94753535-db8d1000-0364-11eb-9b49-ce51dd5d7ab2.png)

![Screenshot_4](https://user-images.githubusercontent.com/54441280/94753537-dc25a680-0364-11eb-963d-1e26760027f1.png)

**Comment by brentvatne** [comment]
does adb work on your machine? eg: if you run `adb devices` from command line what do you see? it seems like there may be an issue with the android sdk installation on your computer

**Comment by SandroFrazaoS** [comment]
Hello Brent

I ran the command

![Screenshot_1](https://user-images.githubusercontent.com/54441280/94754070-91a52980-0366-11eb-877b-842e613939ec.png)

![Screenshot_2](https://user-images.githubusercontent.com/54441280/94754074-9538b080-0366-11eb-8b97-2164b9c826f2.png)

**Comment by brentvatne** [comment]
perhaps this will be helpful: https://stackoverflow.com/questions/36841461/error-android-emulator-gets-killed

if not, try googling that error for similar issues

**Comment by Ivaylo-Iv** [comment]
Hi guys, today I experience the same issue but I have fixed it very easily. Try the solution it might work for you too.
Many developers have wsl or windows subsystem for Linux and it is using a VM and I found it blocks the android sim and causes the error "Error running adb". The solution I found was to open my PowerShell and type "wsl --shutdown". After rerunning the script for android emulation from expo it worked like a charm. Hope I helped.
